### PR TITLE
Added Russian translations to Translator and DataTable components

### PR DESCRIPTION
**Pull request by lopatinas**

### CODE REVIEW

File: assets/components/Translator.js [hunk 1]
```diff
@@ -26,6 +26,19 @@ const TRANS = {
...
     },
+
+    ru: {
+        'row_selected': {
+            '_': '%c% выбрано. <a href data-tag="dt:unselectall">Сбросить</a>'
+        },
+        'cancel': 'Отменить',
+        'confirm': 'Подтвердить',
+        'unauthorized_error': 'Вы отключены, обновите страницу',
+        'forbidden_error': 'У Вас нет прав не выполнение этого действия',
```
Comment (slimus) [inline assets/components/Translator.js]:
```suggestion
        'forbidden_error': 'У Вас нет прав на выполнение этого действия',
```

File: assets/components/Translator.js [hunk 2]
```diff
@@ -26,6 +26,19 @@ const TRANS = {
...
+        'notfound_error': 'Не удается подключиться к серверу',
```
Comment (slimus) [inline assets/components/Translator.js]:
```suggestion
        'notfound_error': 'Не удаётся подключиться к серверу',
```

File: assets/components/Translator.js [hunk 3]
```diff
@@ -26,6 +26,19 @@ const TRANS = {
...
+        'other_error' : 'Произошла ошибка',
+        'loading_data_error': 'Не удается загрузить данные'
```
Comment (slimus) [inline assets/components/Translator.js]:
```suggestion
        'loading_data_error': 'Не удаётся загрузить данные'
```

**Review by slimus — CHANGES_REQUESTED** [review]

File: assets/components/Translator.js (hunk 1 above)
Comment (lopatinas) [reply assets/components/Translator.js]: Fixed

File: assets/components/Translator.js (hunk 2 above)
Comment (lopatinas) [reply assets/components/Translator.js]: Fixed

File: assets/components/Translator.js (hunk 3 above)
Comment (lopatinas) [reply assets/components/Translator.js]: Fixed

**Pull request merged**